### PR TITLE
Update to scoped packages

### PR DESCRIPTION
**Pull request by agubler**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue

**Description:**

Update `package.json` and all dependency references to use `@dojo` namespace.

Related to https://github.com/dojo/meta/issues/129

### CODE REVIEW

File: todo-mvc/src/index.ts [hunk 1]
```diff
@@ -2,16 +2,16 @@
...
-		{ name: 'dojo-app', location: 'node_modules/dojo-app' },
-		{ name: 'dojo-dom', location: 'node_modules/dojo-dom' },
-		{ name: 'dojo-compose', location: 'node_modules/dojo-compose' },
-		{ name: 'dojo-routing', location: 'node_modules/dojo-routing' },
-		{ name: 'dojo-core', location: 'node_modules/dojo-core' },
-		{ name: 'dojo-shim', location: 'node_modules/dojo-shim' },
-		{ name: 'dojo-has', location: 'node_modules/dojo-has' },
-		{ name: 'dojo-stores', location: 'node_modules/dojo-stores' },
-		{ name: 'dojo-widgets', location: 'node_modules/dojo-widgets' },
+		{ name: '@dojo/actions', location: 'node_modules/@dojo/actions' },
```
Comment (matt-gadd) [inline todo-mvc/src/index.ts]: think these lot can be removed in favour of

```
{ name: '@dojo', location: 'node_modules/@dojo },
```

File: todo-mvc/src/index.ts [hunk 2]
```diff
@@ -2,16 +2,7 @@
...
-		{ name: 'dojo-app', location: 'node_modules/dojo-app' },
-		{ name: 'dojo-dom', location: 'node_modules/dojo-dom' },
-		{ name: 'dojo-compose', location: 'node_modules/dojo-compose' },
-		{ name: 'dojo-routing', location: 'node_modules/dojo-routing' },
-		{ name: 'dojo-core', location: 'node_modules/dojo-core' },
-		{ name: 'dojo-shim', location: 'node_modules/dojo-shim' },
-		{ name: 'dojo-has', location: 'node_modules/dojo-has' },
-		{ name: 'dojo-stores', location: 'node_modules/dojo-stores' },
-		{ name: 'dojo-widgets', location: 'node_modules/dojo-widgets' },
+		{ name: '@dojo', location: 'node_modules/@dojo' },
```
Comment (dylans) [inline todo-mvc/src/index.ts]: Is this the kitchensink package?

File: todo-mvc/tests/intern.ts [hunk 3]
```diff
@@ -55,16 +55,7 @@ export const loaderOptions = {
...
-		{ name: 'dojo-app', location: 'node_modules/dojo-app' },
-		{ name: 'dojo-compose', location: 'node_modules/dojo-compose' },
-		{ name: 'dojo-core', location: 'node_modules/dojo-core' },
-		{ name: 'dojo-dom', location: 'node_modules/dojo-dom' },
-		{ name: 'dojo-has', location: 'node_modules/dojo-has' },
-		{ name: 'dojo-routing', location: 'node_modules/dojo-routing' },
-		{ name: 'dojo-shim', location: 'node_modules/dojo-shim' },
-		{ name: 'dojo-stores', location: 'node_modules/dojo-stores' },
-		{ name: 'dojo-widgets', location: 'node_modules/dojo-widgets' },
+		{ name: '@dojo', location: 'node_modules/@dojo' },
```
Comment (dylans) [inline todo-mvc/tests/intern.ts]: Same question, didn't realize or remember that we could then just do @dojo... if so, very cool, I should update the blog post about it.

File: todo-mvc/src/index.ts (hunk 2 above)
Comment (agubler) [inline todo-mvc/src/index.ts]: Nope it is just mapping `@dojo` to the `node_modules/@dojo` directory

**Pull request merged**